src/key.h

```diff
@@ -37,8 +37,8 @@ class CKey
      * script supports up to 75 for single byte push
      */
     static_assert(
-        PRIVATE_KEY_SIZE >= COMPRESSED_PRIVATE_KEY_SIZE,
-        "COMPRESSED_PRIVATE_KEY_SIZE is larger than PRIVATE_KEY_SIZE");
+        PRIVATE_KEY_SIZE > COMPRESSED_PRIVATE_KEY_SIZE,
+        "COMPRESSED_PRIVATE_KEY_SIZE should be smaller than PRIVATE_KEY_SIZE");
```

Comment:
> should be smaller
Compressed private keys are indeed larger than uncompressed ones.

Reply:
This test is relative to the *_SIZE constant values, which are defined above and do have this relationship.

Reply:
Ah I see, should've expanded the lines :)